styles/styles.css

```diff
@@ -333,16 +364,30 @@ main .article-content-wrapper h2:first-of-type {
   margin: unset;
 }
 
+main .section .column-content-top-border > div > div h1 {
+  font-size: 96px;
+  line-height: 120%;
+  letter-spacing: 0;
+  font-weight: 300;
+  margin: 0;
+  }
+
+main .columns.row-header div > div > h2,
```

Comment:
same Q here. Is this specificity required?

Reply:
Same answer -- I have not run into its use yet.